src/frontend/index.html

```diff
@@ -54,216 +60,111 @@
-              through the cites of the word in classical literature, discovered the undoubtable
-              source. Lorem Ipsum comes from sections 1.10.32 and 1.10.33 of "de Finibus Bonorum et
-              Malorum" (The Extremes of Good and Evil) by Cicero, written in 45 BC. This book is a
-              treatise on the theory of ethics, very popular during the Renaissance. The first line
-              of Lorem Ipsum, "Lorem ipsum dolor sit amet..", comes from a line in section 1.10.32.
-              The standard chunk of Lorem Ipsum used since the 1500s is reproduced below for those
-              interested. Sections 1.10.32 and 1.10.33 from "de Finibus Bonorum et Malorum" by
-              Cicero are also reproduced in their exact original form, accompanied by English
-              versions from the 1914 translation by H. Rackham.
-            </p>
-
-            <h3>Heading 3</h3>
+        <!-- Dynamically loaded content goes here -->
+      </div>
+    </div>
 
-            <p>
-              Contrary to popular belief, Lorem Ipsum is not simply random text. It has roots in a
-              piece of classical Latin literature from 45 BC, making it over 2000 years old. Richard
-              McClintock, a Latin professor at Hampden-Sydney College in Virginia, looked up one of
-              the more obscure Latin words, consectetur, from a Lorem Ipsum passage, and going
-              through the cites of the word in classical literature, discovered the undoubtable
-              source. Lorem Ipsum comes from sections 1.10.32 and 1.10.33 of "de Finibus Bonorum et
-              Malorum" (The Extremes of Good and Evil) by Cicero, written in 45 BC. This book is a
-              treatise on the theory of ethics, very popular during the Renaissance. The first line
-              of Lorem Ipsum, "Lorem ipsum dolor sit amet..", comes from a line in section 1.10.32.
-              The standard chunk of Lorem Ipsum used since the 1500s is reproduced below for those
-              interested. Sections 1.10.32 and 1.10.33 from "de Finibus Bonorum et Malorum" by
-              Cicero are also reproduced in their exact original form, accompanied by English
-              versions from the 1914 translation by H. Rackham.
-            </p>
-          </section>
-        </article>
+    <div class="modal-container">
+      <div class="modal-panel">
+        <h1>Title</h1>
+        <section>
+          asdasdsa
+        </section>
       </div>
     </div>
 
-    <script>
-      // CODE TO TOGGLE THE MOBILE MENU DRAWER
-      document.querySelector('#mobile-burger').onclick = function(e) {
-        let nav = document.querySelector('nav');
-        nav.style.transform = 'translateX(-200px)';
-        nav.style.boxShadow = '0px 0px 100px var(--box-shadow-color)';
+    <!-- SCRIPTS -->
 
-        let close = document.querySelector('#mobile-burger-close');
+    <script>
+      $(() => {
```

Comment:
JavaScript in `.js` files please.

Reply:
Moved to main.js, would appreciate any resources on how to keep code in the front end in modular files as I was struggling with that